src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -501,6 +501,45 @@ public object[] Geometry()
             return converted.ToArray();
         }
 
+        /// <summary>
+        /// Gets all elements hosted by the supplied element
+        /// </summary>
+        /// <param name="includeOpenings">Include rectangular openings in output</param>
+        /// <param name="includeShadows">Include shadows in output</param>
+        /// <param name="includeEmbeddedWalls">Include embedded walls in output</param>
+        /// <param name="includeSharedEmbeddedInserts">Include shared embedded elements in output</param>
+        /// <returns>Hosted Elements</returns>
+        public List<Element> GetHostedElements(bool includeOpenings = false,
+                                           bool includeShadows = false,
+                                           bool includeEmbeddedWalls = false,
+                                           bool includeSharedEmbeddedInserts = false)
+        {
+            if (this.InternalElement == null)
+                throw new NullReferenceException();
+            if (Document == null)
+                throw new NullReferenceException();
```

Comment:
we should include a message for these throws since these inputs are hidden from the user, for example `Cannot get hosted elements for a null Element`